edx_exams/settings/local.py

```diff
@@ -151,8 +151,17 @@
 EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL = None
 EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS = None
 
-# The EVENT_BUS_PRODUCER_CONFIG Django setting introduced in openedx-events 8.6.0 does not work properly.
-# In order to unblock development, EVENT_BUS_PRODUCER_CONFIG should be set to the empty object. Signal handlers
-# should be added as before (i.e. versions <8.6.0).
-# Once openedx-events 8.6.0 is corrected, this Django setting should be set.
-EVENT_BUS_PRODUCER_CONFIG = {}
+attempt_submitted_event_config = EVENT_BUS_PRODUCER_CONFIG['org.openedx.learning.exam.attempt.submitted.v1']
```

Comment:
[question] could you clarify the difference between the config in local and base? Wondering why it is set in both places, but I might be missing something. Thanks!

Reply:
in `base.py` the event producers are disabled, whereas for local development we want them enabled.

Reply:
Oh I understand now with the other PR. This is set in edx-internal but we still need the settings set locally.